tests/testthat/test-length.R

```diff
@@ -1,15 +1,209 @@
-suppressWarnings({
-  library(haven)
-  library(readxl)
+#' Test `xportr_length()` function
+#'
+#' Tests will check for:
+#'  * Errors
+#' * Result of call will create `SASlength` attribute (`width` for each
+#' variable)
+#'
+#' Helper functions / data (from ´test/testthat/helper-length.R´):
+#'  * \code{minimal_adsl} : minimal data frame with ADSL sample
+#'  * \code{minimal_length_metadata} : minimal metadata used in
+#'    `xport_length()`. It contains dataset (domain), variable names and length
+#'  * \code{expect_attr_width(result, metadata_length)} : support function to
+#'    test the data.frame modifications being done in `xportr_length()`
```

Comment:
Sounds good to me! 

@bms63 Are you envisioning a "minimal metadata" and "minimal dataset" and including it in the scope of this PR?

Or a function that takes a bunch of logical parameters and builds that minimal example (`length`, `label`, `type`, ...)?

Reply:
I know some other packages have some minimal data in their `data/` folder to use for testing and examples. I think that would be a great improvement to the testing and examples instead of using slightly different everything.